app/models/user.rb

```diff
@@ -9,9 +9,9 @@ class User < ApplicationRecord
   # https://github.com/heartcombo/devise/blob/master/lib/devise/models/validatable.rb
   validates_uniqueness_of :email, scope: :company_id
   validates_format_of     :email, with: email_regexp, allow_blank: true, if: :email_changed?
-  validates_presence_of     :password, if: :password_required?
-  validates_confirmation_of :password, if: :password_required?
-  validates_length_of       :password, within: password_length, allow_blank: true
+  # validates_presence_of     :password, if: :password_required?
```

Comment:
remove if not required

Reply:
It was troubling while manual password change. Now it is fine to have these validations.